app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -56,6 +56,7 @@ class SelectionInteractionViewModel private constructor(
     computeChoiceItems(choiceSubtitledHtmls, hasConversationView, this)
 
   private val isAnswerAvailable = ObservableField(false)
+  val selectedItemText = ObservableField("Please select atleast one choice.")
```

Comment:
We need to use string resources for these strings, instead, since they need to be translatable.